src/core/operators/QueryOperators.ts

```diff
@@ -9,19 +11,32 @@ const parseAndFind = (
     if (Object.keys(query).length === 0) {
       return docs;
     }
-    const filteredDocs = [];
+    var filteredDocs = [];
```

Comment:
Please don't use var, it's against es6 standard.

Use const for non-changing values and let for values that change later.

Reply:
I'll update that. I'll take a bit of time for me to get used to this :sweat_smile:

Reply:
done.